### ISSUE TITLE
Support for Llama3 Fine Tuned Function Calling Models 

### ISSUE THREAD

**Issue by Deepansharora27** [llmware-ai/llmware#996]
Hi, 

Apart from using the SLIM Models for Function Calling, Can I make Use Llama3 Derivates  Of Fined Tuned Function Call Models ?

**Comment by King10T** [comment]
Yes, you can use fine-tuned function calling models derived from LLaMA 3 as long as those models have been specifically trained or fine-tuned for function calling tasks.

**Comment by Ikhlano** [comment]
What specific functions are you looking for Llama3 to call?"
